src/api/public/apidocs-new/components/responses/bad_request_error.yaml

```diff
@@ -14,3 +14,7 @@ content:
         value:
           code: err_register_save
           summary: "Could not save the registration, details: Login is the name of an already existing user and Password can't be blank."
+      Validation Failed:
+        value:
+          code: validation_failed
+          summary: "status_message validation error: 2:0: ERROR: Element '<some element>': No matching global declaration available for the validation root."
```

Comment:
The component `responses/bad_request_error.yaml` is also used in the `POST /person/{login}` endpoint.

Does this "Validation Failed" error introduced for status messages apply also for the person endpoint? If not, I wouldn't add the "Validation Failed" example into the `responses/bad_request_error.yaml` component.